Release/0.3.0

demogar opened the pull request.

demogar committed:
docs(changeset): add font family tokens

demogar committed:
RELEASING: Releasing 4 package(s)

Releases:
  @productplan/atlas-docs@0.3.0
  @productplan/atlas-tokens@0.2.0
  @productplan/atlas-web-components@0.3.0
  @productplan/atlas@0.3.0

[skip ci]

demogar committed:
chore(global): remove changeset for github

nullcitizen commented on packages/atlas-tokens/package.json:
```diff
@@ -2,7 +2,7 @@
   "name": "@productplan/atlas-tokens",
   "homepage": "https://github.com/ProductPlan/atlas",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
I have ruined everything, and 0.2.0 was already published. this'll need a version bump

demogar committed:
docs(changeset): - add font family
- add author to package.json
- add license
- add shields to repo

demogar committed:
RELEASING: Releasing 4 package(s)

Releases:
  @productplan/atlas-docs@0.3.1
  @productplan/atlas@0.3.1
  @productplan/atlas-tokens@0.2.1
  @productplan/atlas-web-components@0.3.1

[skip ci]
